www/include/monitoring/objectDetails/xml/serviceSendCommand.php

```diff
@@ -60,47 +62,66 @@
 $pearDB = new CentreonDB();
 $hostObj = new CentreonHost($pearDB);
 $svcObj = new CentreonService($pearDB);
-$host_id = $_GET["host_id"];
-$svc_id = $_GET["service_id"];
-$poller = $hostObj->getHostPollerId($host_id);
-$cmd = $_GET["cmd"];
-$sid = session_id();
-$act_type = $_GET["actiontype"];
+
+$hostId = filter_var(
+    $_GET['host_id'] ?? false,
+    FILTER_VALIDATE_INT
+);
+
+$serviceId = filter_var(
+    $_GET['service_id'] ?? false,
+    FILTER_VALIDATE_INT
+);
+
+$pollerId = $hostObj->getHostPollerId($hostId);
```

Comment:
getHostPollerId won't check if the argument is false or not
you need to add the check in the getHostPollerId() method

Reply:
**Done** Thanks :+1: